### PR TITLE
feat(stryker-elements): Add Storybook

### PR DESCRIPTION
**Pull request by lex-funy**
closes: #542

### CODE REVIEW

File: packages/stryker-elements/package.json [hunk 1]
```diff
@@ -39,14 +39,25 @@
...
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006"
   },
   "peerDependencies": {
     "lit": "^2.7.6"
   },
   "devDependencies": {
+    "@chromatic-com/storybook": "^1.2.19",
```
Comment (xandervedder) [inline packages/stryker-elements/package.json]: Do we need this dependency?

File: packages/stryker-elements/package.json [hunk 2]
```diff
@@ -39,14 +39,25 @@
...
+    "@storybook/addon-essentials": "^8.0.0",
+    "@storybook/addon-links": "^8.0.0",
+    "@storybook/blocks": "^8.0.0",
+    "@storybook/manager-api": "^8.0.0",
+    "@storybook/test": "^8.0.0",
+    "@storybook/theming": "^8.0.0",
+    "@storybook/web-components": "^8.0.0",
+    "@storybook/web-components-vite": "^8.0.0",
```
Comment (xandervedder) [inline packages/stryker-elements/package.json:56]: Can we pin these dependencies? E.g.: `^8.0.0` -> `8.0.0` (or whatever is latest)

File: packages/stryker-elements/.storybook/main.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import type { StorybookConfig } from "@storybook/web-components-vite";
+
+const config: StorybookConfig = {
+  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@chromatic-com/storybook",
```
Comment (xandervedder) [inline packages/stryker-elements/.storybook/main.ts:8]: Remove this

**Review by xandervedder — CHANGES_REQUESTED** [review]
Could you check if these stories are in the `dist` folder when building?

File: packages/stryker-elements/package.json (hunk 1 above)
Comment (lex-funy) [reply packages/stryker-elements/package.json]: No

File: packages/stryker-elements/package.json (hunk 2 above)
Comment (lex-funy) [reply packages/stryker-elements/package.json]: 8.0.0 is the latest version and has been pinned.

File: packages/stryker-elements/.storybook/main.ts (hunk 3 above)
Comment (lex-funy) [reply packages/stryker-elements/.storybook/main.ts]: 👍

**Review by xandervedder — APPROVED** [review]

**Pull request merged**